libc-test/build.rs

```diff
@@ -145,11 +145,41 @@ fn main() {
     // Avoid unnecessary re-building.
     println!("cargo:rerun-if-changed=build.rs");
 
+    if std::fs::exists("../target/src-hotfix").unwrap() {
```

Comment:
Sorry, I just noticed one more thing: I think these need to use `CARGO_TARGET_DIR` since if that is overridden then we shouldn't write to `../target`.

Reply:
I used `OUT_DIR` instead, since I think it is more appropriate and Cargo won't set `CARGO_TARGET_DIR` if the user does not define it.